templates/server.templ

```diff
@@ -179,21 +179,6 @@ func (s *Server) SetOnClosed(f func(*frisbee.Async, error)) error {
                             srv.CloseSend()
                         }
                     }()
-                {{ if not (or $method.IsStreamingServer $method.IsStreamingClient) -}}
```

Comment:
This isn't necessary because this entire section is wrapped in an `if or` - so this `if not (or` will never get executed. This seems to be an erroneous copy-paste, since this exact block is repeated below (without being wrapped in the `if or`